omod/src/main/java/org/openmrs/module/registrationapp/page/controller/RegisterPatientPageController.java

```diff
@@ -56,6 +61,17 @@ public void addModelAttributes(PageModel model, Patient patient, AppDescriptor a
         model.addAttribute("enableOverrideOfAddressPortlet",
                 Context.getAdministrationService().getGlobalProperty("addresshierarchy.enableOverrideOfAddressPortlet", "false"));
         model.addAttribute("breadcrumbOverride", breadcrumbOverride);
+
+        // Relationship Types to exclude (Doctor/Patient and Supervisor/Supervisee)
```

Comment:
I'm not sure of that. It was mentioned in one of the `app.json` files in some module.